src/Microsoft.AspNetCore.Server.Kestrel/Internal/Infrastructure/MemoryPool.cs

```diff
@@ -64,16 +65,33 @@ public class MemoryPool : IDisposable
         /// Called to take a block from the pool.
         /// </summary>
         /// <returns>The block that is reserved for the called. It must be passed to Return when it is no longer being used.</returns>
+#if DEBUG
+        public MemoryPoolBlock Lease([CallerMemberName] string memberName = "",
+                             [CallerFilePath] string sourceFilePath = "",
+                             [CallerLineNumber] int sourceLineNumber = 0)
```

Comment:
Another one of my super nits :) Put the first arg in its own line, and indent all args with 4 spaces only.